### PR TITLE
Added media queries to embedded examples

### PR DESCRIPTION
**Pull request by jomar-honsi**
Added CSS media queries to examples that rendered messy on narrow screens.

### CODE REVIEW

File: samples/dashboards/basic/your-first-dashboard/demo.css [hunk 1]
```diff
@@ -4,3 +4,9 @@
 #dashboard-col-0 {
     text-align: center;
 }
+
+@media screen and (max-width: 680px) {
```
Comment (sebastianbochan) [inline samples/dashboards/basic/your-first-dashboard/demo.css:8]: I prefer to use the `576px` that is standardized.

Comment (jomar-honsi) [reply samples/dashboards/basic/your-first-dashboard/demo.css:8]: Agree.

**Review by stitot — CHANGES_REQUESTED** [review]
Change all too 576px

**Review by sebastianbochan — APPROVED** [review]

**Review by stitot — APPROVED** [review]
Thanks!

**Pull request merged**